.eslintrc.js

```diff
@@ -13,7 +13,6 @@ module.exports = {
     'eslint:recommended',
     'plugin:@typescript-eslint/eslint-recommended',
     'plugin:@typescript-eslint/recommended',
-    'prettier/@typescript-eslint',
```

Comment:
This plugin is now inside the `prettier` itself so we don't need to provide it anymore.